nltk/tree/prettyprinter.py

```diff
@@ -207,7 +207,7 @@ def dumpmatrix():
             raise ValueError("All leaves must be integer indices.")
         if len(leaves) != len(set(leaves)):
             raise ValueError("Indices must occur at most once.")
-        if not all(0 <= n < len(sentence) for n in leaves):
+        if not all(n in range(0, len(sentence)) for n in leaves):
```

Comment:
hey mohaned! this actually felt a bit better before. plus now it's constructing a list for each element in `leaves` which feels a bit inefficient. what do you think?

Reply:
Yes, you are right. 

Thanks.